### PR TITLE
remove some incompatible flags for Bazel 0.25.2

### PR DESCRIPTION
**Pull request by evie404**
Been experimenting which incompatible flags we don't need. This seems to be the set we can remove without upgrading some deps like grpc, which itself is quite gnarly zzz

build passed in: https://travis-ci.org/vsco/rules_protobuf/builds/586057629

### CODE REVIEW

File: protobuf/rules.bzl [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
+load("//protobuf:internal/proto_http_archive.bzl", _proto_http_archive = "proto_http_archive")
+load("//protobuf:internal/proto_dependencies.bzl", _proto_dependencies = "proto_dependencies")
+
+proto_compile = _proto_compile
+proto_language = _proto_language
+proto_language_deps = _proto_language_deps
+proto_repositories = _proto_repositories
+github_archive = _github_archive
+proto_http_archive = _proto_http_archive
+proto_dependencies = _proto_dependencies
```
Comment (akakitani) [inline protobuf/rules.bzl:14]: are all these changes addressing the `no_transitive_loads` flag?

File: cpp/deps.bzl [hunk 2]
```diff
@@ -13,6 +13,8 @@ DEPS = {
         "rule": "http_archive",
         # master-with-bazel Fri Sep 01 15:09:13 2017 +0000
         "url": "https://boringssl.googlesource.com/boringssl/+archive/886e7d75368e3f4fab3f4d0d3584e4abfc557755.tar.gz",
+        # don't attempt to checksum this as it does not appear to be stable
```
Comment (akakitani) [inline cpp/deps.bzl:16]: wat

**Review by akakitani — APPROVED** [review]
thank you! 

Were you able to do something with `bazelisk --migrate` to figure out the set of compatible flags btw? Or did you have to go through the flags manually

**Comment by evie404** [comment]
@akakitani figured out those manually zzz

File: cpp/deps.bzl (hunk 2 above)
Comment (evie404) [reply cpp/deps.bzl:16]: yeah idk it was super weird :s

File: protobuf/rules.bzl (hunk 1 above)
Comment (evie404) [reply protobuf/rules.bzl:14]: yes. otherwise we were implicitly using those internal rules

**Pull request merged**